### DISCUSSION TITLE
error 500 after server restore to new server

### DISCUSSION THREAD

**Discussion by rwcloudridgeworks** [amidaware/tacticalrmm#1290] [General]
**Server Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04
 - Browser: [e.g. chrome
 - RMM Version (as shown in top left of web UI):

**Installation Method:**
  - [ x] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI):
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**

**To Reproduce**
go to rmm.rwcloud.co.uk and try and login get the 400 bad cred 

I have checked server time , DNS, updates, all fine 

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by dinger1986** [comment]
this is a discussion not an issue as it is an issue directly with your installation.

As suggested to neomatrix666 on discord please try and restore again to a new server.

Sending an email to help [@] amidaware.com with info would maybe help if you are struggling to resolve the issue
